.github/workflows/reusable-node-test.yml

```diff
@@ -61,8 +60,7 @@ jobs:
 
       - name: Node Test For Node20
         run: |
-          pnpm run --filter rolldown-tests test:main 
-          pnpm run --filter rolldown-tests test:watcher
+          pnpm run --filter rolldown-tests test:main && pnpm run --filter rolldown-tests test:watcher
```

Comment:
Update it to make sure ci failed if `test:main` failed, see https://github.com/rolldown/rolldown/actions/runs/12986599769/job/36213777523?pr=3436